README.md

```diff
@@ -1 +1,57 @@
-# vcmi-dependencies
\ No newline at end of file
+# VCMI Dependencies
+
+This repository contains prebuilt Conan dependencies for vcmi. These dependencies are primary used by Github CI and can be used to avoid building dependencies locally by developers.
+
+Current flow to update dependencies:
+- push new version of conanfile and/or CI settings to vcmi/vcmi repository, to branch 'update_prebuilts'
+- run CI in this repository to generate new packages
+- create new release from generated artifacts
+- open PR in vcmi/vcmi with changes to conanfile, CI, new URL to dependencies package and whatever changes need to be done to source code to use newer package (if any)
+
+# TODO List
+
+- Find a way to set up an Artifactory on our server or find somebody willing to host it for us (e.g. JFrog) and deprecate this repository
```

Comment:
there's also one more option, although not really recommended - https://docs.conan.io/2/reference/conan_server.html

I tested it in v1 on our server - seems to work, at least I was able to upload and download packages